presto-main/src/main/java/com/facebook/presto/cost/ProjectStatsRule.java

```diff
@@ -53,7 +53,8 @@ public ProjectStatsRule(ScalarStatsCalculator scalarStatsCalculator, StatsNormal
     {
         PlanNodeStatsEstimate sourceStats = statsProvider.getStats(node.getSource());
         PlanNodeStatsEstimate.Builder calculatedStats = PlanNodeStatsEstimate.builder()
-                .setOutputRowCount(sourceStats.getOutputRowCount());
+                .setOutputRowCount(sourceStats.getOutputRowCount())
+                .setConfident(sourceStats.isConfident());
```

Comment:
Based on the offline discussion: Let's set it to confident only when we are confident no columns have changed the cardinallity

Reply:
Done